src/code/z_en_item00.c

```diff
@@ -1188,9 +1168,7 @@ s32 func_800A8150(s32 index) {
     return D_801AE194[index];
 }
 
-u8 D_801AE214[32] = {
-    0, 0, 0, 0, 0, 0, 0, 0, 1, 1, 0, 0, 1, 1, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
-};
+u8 D_801AE214[32] = { 0, 0, 0, 0, 0, 0, 0, 0, 1, 1, 0, 0, 1, 1, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 };
```

Comment:
I think I very slightly prefer this is it was previously with the comma, but it's not a big deal